lib/active_admin_import/model.rb

```diff
@@ -5,8 +7,9 @@ class Model
     include ActiveModel::Validations
     include ActiveModel::Validations::Callbacks
 
-    validates :file, presence: {message: Proc.new { I18n.t('active_admin_import.no_file_error') }},
-              unless: proc { |me| me.new_record? }
+    validates :file, presence: {
+        message: proc { I18n.t("active_admin_import.no_file_error") }
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.